geos/entrypoints_test.go

```diff
@@ -927,3 +935,19 @@ func TestCoverageUnion(t *testing.T) {
 		})
 	}
 }
+
+func TestCoverageSimplifyVW(t *testing.T) {
+	input := geom.NewGeometryCollection(
+		[]geom.Geometry{
+			geomFromWKTFile(t, "testdata/coverage_simplify_input_birchgrove.wkt"),
+			geomFromWKTFile(t, "testdata/coverage_simplify_input_balmain.wkt"),
+		},
+	)
+	got, err := geos.CoverageSimplifyVW(input.AsGeometry(), 0.001, false)
+	if err != nil && strings.Contains(err.Error(), "unsupported") {
```

Comment:
I've used string comparison for a sentinel error, which is not ideal.

I plan to fix this using the same treatment as in https://github.com/peterstace/simplefeatures/pull/616 (there are a few instances in this package that could benefit from it, so I'll do them all at once).

Reply:
https://github.com/peterstace/simplefeatures/pull/624/files